main/options/options.cc

```diff
@@ -17,62 +17,68 @@
+    {"rewrite-tree-raw", &Printers::RewriterTreeRaw, false},
+    {"index-tree", &Printers::IndexTree},
+    {"index-tree-raw", &Printers::IndexTreeRaw},
+    {"name-tree", &Printers::NameTree},
+    {"name-tree-raw", &Printers::NameTreeRaw},
+    {"resolve-tree", &Printers::ResolveTree},
+    {"resolve-tree-raw", &Printers::ResolveTreeRaw},
+    {"flatten-tree", &Printers::FlattenTree},
+    {"flatten-tree-raw", &Printers::FlattenTreeRaw},
+    {"ast", &Printers::AST},
+    {"ast-raw", &Printers::ASTRaw},
+    {"cfg", &Printers::CFG},
+    {"cfg-raw", &Printers::CFGRaw},
+    {"cfg-text", &Printers::CFGText},
+    {"symbol-table", &Printers::SymbolTable},
+    {"symbol-table-raw", &Printers::SymbolTableRaw},
+    {"symbol-table-json", &Printers::SymbolTableJson},
+    {"symbol-table-proto", &Printers::SymbolTableProto},
     {"symbol-table-messagepack", &Printers::SymbolTableMessagePack, true, false},
-    {"symbol-table-full", &Printers::SymbolTableFull, true},
-    {"symbol-table-full-raw", &Printers::SymbolTableFullRaw, true},
-    {"symbol-table-full-json", &Printers::SymbolTableFullJson, true},
-    {"symbol-table-full-proto", &Printers::SymbolTableFullProto, true},
+    {"symbol-table-full", &Printers::SymbolTableFull},
+    {"symbol-table-full-raw", &Printers::SymbolTableFullRaw},
+    {"symbol-table-full-json", &Printers::SymbolTableFullJson},
+    {"symbol-table-full-proto", &Printers::SymbolTableFullProto},
     {"symbol-table-full-messagepack", &Printers::SymbolTableFullMessagePack, true, false},
-    {"file-table-json", &Printers::FileTableJson, true},
-    {"file-table-proto", &Printers::FileTableProto, true},
+    {"file-table-json", &Printers::FileTableJson},
+    {"file-table-proto", &Printers::FileTableProto},
     {"file-table-messagepack", &Printers::FileTableMessagePack, true, false},
-    {"file-table-full-json", &Printers::FileTableFullJson, true},
-    {"file-table-full-proto", &Printers::FileTableFullProto, true},
+    {"file-table-full-json", &Printers::FileTableFullJson},
+    {"file-table-full-proto", &Printers::FileTableFullProto},
     {"file-table-full-messagepack", &Printers::FileTableFullMessagePack, true, false},
-    {"missing-constants", &Printers::MissingConstants, true},
-    {"autogen", &Printers::Autogen, true},
-    {"autogen-msgpack", &Printers::AutogenMsgPack, true},
-    {"autogen-classlist", &Printers::AutogenClasslist, true},
+    {"missing-constants", &Printers::MissingConstants},
+    {"autogen", &Printers::Autogen},
+    {"autogen-msgpack", &Printers::AutogenMsgPack},
+    {"autogen-classlist", &Printers::AutogenClasslist},
     {"autogen-autoloader", &Printers::AutogenAutoloader, true, false},
-    {"autogen-subclasses", &Printers::AutogenSubclasses, true},
-    {"package-tree", &Printers::Packager},
+    {"autogen-subclasses", &Printers::AutogenSubclasses},
+    {"package-tree", &Printers::Packager, false},
+    {"minimized-rbi", &Printers::MinimizeRBI},
```

Comment:
This `minimized-rbi` is the new option.

The rest are just because I didn't like the defaults for `PrintOptions` and wanted to flip them, which I did in `1506a85d8fca5a04e6de15c026e2a4d50e12c7be` if you want to see the minimal diff for that.